acl/acl.go

```diff
@@ -410,6 +410,35 @@ func (a *ACL) AllowNodePool(pool string) bool {
 	return !capabilities.Check(PolicyDeny)
 }
 
+// AllowAnyNodePool returns true if any operation is allowed in at least one
+// node pool.
+//
+// This is a very loose check and is expected that callers perform more precise
+// verification later.
+func (a *ACL) AllowAnyNodePool() bool {
```

Comment:
Maybe this could be `AllowNodePoolSearch` so that it's clear when people autocomplete in their editors exactly the difference? I think we did that for Variables.

Reply:
Ah nice, I didn't notice that. I think I was going off the namespace methods.